saml2/saml2-service-provider/src/main/java/org/springframework/security/saml2/provider/service/web/authentication/Saml2AuthenticationRequestResolver.java

```diff
@@ -29,6 +29,8 @@
  */
 public interface Saml2AuthenticationRequestResolver {
 
+	String DEFAULT_SAML_AUTHENTICATION_REQUEST_URL = "/saml2/authenticate/{registrationId}";
```

Comment:
Please call this `DEFAULT_AUTHENTICATION_REQUEST_URI`, removing `SAML` and changing `URL` to `URI`. This follows more closely the same name found in `OAuth2AuthorizationRequestRedirectFilter`.